_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,361 @@
+
+- **Coinswap:** Aviv Milner presented on coinswap at the Wasabi Research
+  Club. He explained the property of covertness and how Chris Belcher’s
+  coinswap proposal (see [Newsletter #100][news100 coinswap]) provides
+  covertness in a manner that other privacy schemes such as
+  [coinjoin][topic coinjoin] fail to do. Milner also went through the
+  motivation for routing coinswaps, namely to address the concern of
+  unwittingly entering into a coinswap with an adversary such a chain
+  surveillance company.  ([transcript][coinswap ts], [video][coinswap
+  vid])
+
+- **Schnorr signatures and multisignatures:** [BIP340][] co-authors Tim
+  Ruffing, Pieter Wuille, and Jonas Nick held a discussion at London
+  Bitcoin Devs on the specification of [schnorr signatures][topic
+  schnorr signatures].  This covered earlier ideas for implementing
+  schnorr signatures in Bitcoin and what the co-authors thought the
+  community should be concerned about when considering a possible future
+  soft fork deployment. Wuille noted that he is most concerned about how
+  schnorr is adopted and what is built using it.  The following day, Tim
+  Ruffing presented on the challenges of multisignature and threshold
+  signature schemes using schnorr signatures (see also [Newsletter
+  #68][news68 ruffing]).  He's been working with
+  collaborators on a speculative [MuSig][topic musig] signature scheme
+  that only requires two rounds of interaction rather than three,
+  without relying on zero knowledge proofs, which would greatly simplify
+  some threshold and multisignature signing workflows.  ([Meeting transcript][socratic ts],
+  [meeting video][socratic vid], [presentation transcript][ruffing ts],
+  [presentation video][ruffing vid], [presentation slides][ruffing
+  slides])
+
+- **Sydney meetup discussion:** A number of Bitcoin and LN developers
+  joined this Sydney meetup to discuss various topics. Ruben Somsen gave
+  a short presentation on Succinct Atomic Swaps (SAS) (see [Newsletter
+  #98][news98 sas]) and how it compares to Chris Belcher’s coinswap
+  proposal.  Another topic was Bitcoin Core's reliance on DNS seeds to
+  find initial peers, whether it's a potential attack vector against newly
+  started nodes, and how work by Matt Corallo and Antoine Riard on
+  allowing [alternative transports][Bitcoin Core #18988] could help
+  mitigate some risks.  Finally, Lloyd Fournier discussed how
+  experimenting with his toy Rust implementation of secp256k1
+  (secp256kfun) led to a small [fix][libsecp256k1 #732] in the ECDSA
+  signature code in the actual secp256k1 library. The transcript was
+  anonymized to encourage open discussion.  ([transcript][sydney ts])
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 1.1.2][hwi 1.1.2]: this release includes support
```

Comment:
Should we note this is a RC?

Reply:
FIxed, thanks!

Reply:
> Should we note this is a RC?

I could be wrong but it looks like the RC was a week ago and this is a release: https://github.com/bitcoin-core/HWI/tags

Reply:
@jonatack Yep: https://github.com/bitcoin-core/HWI/releases/tag/1.1.2